docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -256,6 +257,28 @@ test('My test', async t => {
 });
 ```
 
+### Filter Predicate
+
+The `filterFn` function used in `filter`, `find`, `parent`, `child` and `sibling` methods is a predicate that is executed on the client side.
+
+This predicate takes the following parameters.
+
+Property | Description
+------ | -----
+`node`  | The current DOM node.
+`idx` | Index of the current node among other nodes returned by the preceding selector.
+`originNode` | Node returned by the preceding selector.
```

Comment:
That's why we have an example below.
I don't know how to describe `originNode` in words without blowing up reader's mind.
Sometimes a brief description and an example do the job better.